nautobot/project-static/css/base.css

```diff
@@ -689,6 +689,20 @@ td span.hover_copy:hover .hover_copy_button {
     margin-right: 0;
     padding: 8px 20px;
     white-space: normal;
+    text-decoration: none;
+    transition: color 200ms ease;
```

Comment:
What are these changes doing? I don't see any difference in the copy button behavior

Reply:
The text-decoration removes the underline on the navbar links (good catch - I removed it from here because it's already covered in another style rule).

The transition makes the color change of text smooth versus immediate.

Reply:
> ... I removed it from here because it's already covered in another style rule...

Actually, there was an instance where the underline was still visible, and this removes it. So keeping this